tests/incremental/11-restart/15-mutex-simple-wrap1.patch

```diff
@@ -1,22 +1,16 @@
+diff --git tests/incremental/11-restart/15-mutex-simple-wrap1.c tests/incremental/11-restart/15-mutex-simple-wrap1.c
+index 790157639..4e9ef494e 100644
 --- tests/incremental/11-restart/15-mutex-simple-wrap1.c
 +++ tests/incremental/11-restart/15-mutex-simple-wrap1.c
-@@ -7,15 +7,15 @@ pthread_mutex_t mutex2 = PTHREAD_MUTEX_INITIALIZER;
-
- void *t_fun(void *arg) {
-   pthread_mutex_lock(&mutex1);
--  myglobal=myglobal+1; // RACE!
-+  myglobal=myglobal+1; // RACE (not enough fuel)
```

Comment:
It would be clearer if the patch still changed the comment like this to make the point.

Reply:
Thanks for the hint. I overlooked that.